locations/spiders/krispy_kreme_jp.py

```
@@ -0,0 +1,187 @@
+from scrapy.spiders import SitemapSpider, Spider
+
+from locations.categories import Categories, Extras, apply_yes_no
+from locations.hours import DAYS, DAYS_JP, OpeningHours, day_range
+from locations.items import Feature
+
+
+class KrispyKremeJPSpider(SitemapSpider, Spider):
+    name = "krispy_kreme_jp"
+    allowed_domains = ["krispykreme.jp"]
+    start_urls = ["https://krispykreme.jp"]
+    item_attributes = {"brand": "Krispy Kreme", "brand_wikidata": "Q1192805", "extras": Categories.FAST_FOOD.value}
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)